go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/google/go-cmp v0.3.0
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
-	github.com/openshift/api v3.9.1-0.20191028134408-7e36eed0d19e+incompatible
+	github.com/openshift/api v0.0.0-20200114154634-6f8de8790817
```

Comment:
Curious as to why the v-part of the version changed so much? no tag?

Reply:
Yeah, no tags